### PR TITLE
Mana curve x cost fix

### LINKED ISSUE #246: Wrapping columns
I believe these shouldn't be allowed to wrap (decks page table). A simple white-space: nowrap should fix this I imagine.

image

### PR DESCRIPTION
**Pull request by lily-pytel**
- Associate -1 cost cards as 1 cost cards (but without abs)
- Remove the ability for most cells in the decks table to wrap - fixes #246

### CODE REVIEW

File: next/lib/deck-stats.js [hunk 1]
```diff
@@ -42,7 +42,16 @@ export const getManaCurveHighchartsSeries = (cards, theme) => {
     Object.values(cards).forEach(c => {
       const { quantity, card } = c;
       const primaryFaction = mainFaction(card);
-      const manaCost = card.mana >= 6 ? '6+' : card.mana.toString();
+
+      let manaCost = card.mana;
+      if (manaCost === -1) {
```
Comment (lily-pytel) [inline next/lib/deck-stats.js]: Too explicit?

Comment (alexmiddeleer) [reply next/lib/deck-stats.js]: No. I might even make it more explicit by adding a constant X_COST

Comment (lily-pytel) [reply next/lib/deck-stats.js]: Can't tell if you're kidding

Comment (lily-pytel) [reply next/lib/deck-stats.js]: But here you go @alexmiddeleer : https://github.com/mythgard-hub/mythgard-hub/pull/248/commits/654674fe0a6223fa2ec7cc937741156169d608e3

**Pull request merged**